packages/perseus-editor/src/widgets/plotter-editor.tsx

```diff
@@ -47,6 +38,15 @@ const editorDefaults = {
     snapsPerLine: 2,
 } as const;
 
+export const plotTypes = [
+    "bar",
+    "line",
+    "pic",
+    "histogram",
+    "dotplot",
+] as const;
+export type PlotType = typeof plotTypes[number];
+
```

Comment:
These are already defined in `Perseus-types.ts` so I _think_ we can reference them through the exported `PerseusPlotterWidgetOptions` (you might need to add it as an export in the root Perseus `index.ts`, but I think that's the way that makes sense).